### PR TITLE
Fix: Changing settings screen pallets colors

### PR DESCRIPTION
**Pull request by D-Rosa99**
#### Description:

I changed the color of settings screen along with all the buttons styles and color

### CODE REVIEW

**Review by RutsSantos — APPROVED** [review]
LGTM

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

File: app/constants/DR/buttonStyles.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import {
```
Comment (iocedano) [inline app/constants/DR/buttonStyles.js:1]: Why we should place in thee constants folder? 
Could we add the style near to the components?

Comment (D-Rosa99) [reply app/constants/DR/buttonStyles.js:1]: this file is global styles, most of the buttons in the project should have  this style

Comment (D-Rosa99) [reply app/constants/DR/buttonStyles.js:1]: They're not for a specific one

File: app/assets/svgs/permissionUnknown.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export default `<svg width="41" height="41" viewBox="0 0 41 41" fill="none" xmlns="http://www.w3.org/2000/svg">
-<circle cx="20.5" cy="20.5" r="20.5" fill="#2434B6"/>
```
Comment (adnanjt) [inline app/assets/svgs/permissionUnknown.js]: is this file a MIT or our asset ? if so can we recreate or change this another way ?

**Review by adnanjt — COMMENTED** [review]
Questions

File: app/assets/svgs/permissionUnknown.js (hunk 2 above)
Comment (D-Rosa99) [reply app/assets/svgs/permissionUnknown.js]: well, we do not need it, so maybe I can just remove the reference where I using it

Comment (adnanjt) [reply app/assets/svgs/permissionUnknown.js]: great

**Review by adnanjt — APPROVED** [review]
LGTM need to resolve test

**Comment by adnanjt** [comment]
Tienes que hacer  yarn test -- -u
para actualizar los snapshots

File: jest/config.js [hunk 3]
```diff
@@ -10,4 +10,7 @@ module.exports = {
     'node_modules',
     'node_modules/(?!(@react-native-community|react-native)/)',
   ],
+  transformIgnorePatterns: [
```
Comment (adnanjt) [inline jest/config.js:13]: @iocedano  DO you think this got any implications? He solved the failing test like this

**Pull request merged**